<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-C34UQS0" failures="0" tests="8" name="Testcase.MP_FreeUpgradePath" time="163.579" errors="1" timestamp="15 Oct 2018 17:09:52 GMT" skipped="0">
  <testcase classname="Testcase.MP_FreeUpgradePath" name="getAllEvents" time="163.570"/>
  <testcase classname="Testcase.MP_FreeUpgradePath" name="e_ClickOnUpgrade" time="0.002">
    <error message="JSONObject[&quot;type&quot;] not found." type="org.json.JSONException">
      <![CDATA[org.json.JSONException: JSONObject["type"] not found.
at org.json.JSONObject.get(JSONObject.java:454)
at Function.F_Mixpanel_API.verifyEventByPropertyValue(F_Mixpanel_API.java:291)
at Testcase.MP_FreeUpgradePath.e_ClickOnUpgrade(MP_FreeUpgradePath.java:219)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- e_ClickOnUpgrade -->
  <testcase classname="Testcase.MP_FreeUpgradePath" name="e_ClickOnBusinessUpgrade" time="0.002"/>
  <testcase classname="Testcase.MP_FreeUpgradePath" name="e_BrandKitUpgrade" time="0.001"/>
  <testcase classname="Testcase.MP_FreeUpgradePath" name="e_ClickOnUpgradeBusinessLink" time="0.001"/>
  <testcase classname="Testcase.MP_FreeUpgradePath" name="e_ClickOnUpgradeFromBrandKit" time="0.001"/>
  <testcase classname="Testcase.MP_FreeUpgradePath" name="intercom_ClickOnUpgrade" time="0.001"/>
  <testcase classname="Testcase.MP_FreeUpgradePath" name="intercom_ClickOnBusinessUpgrade" time="0.001"/>
</testsuite> <!-- Testcase.MP_FreeUpgradePath -->
